app/models/grda_warehouse/tasks/README_FOR_IDENTIFY_DUPLICATES.md

```diff
@@ -0,0 +1,149 @@
+# Duplicate Client Identification Algorithm
+
+The `IdentifyDuplicates` deduplicates client records by identifying and merging clients that represent the same person across different data sources.
+
+## Architecture Overview
+
+The system operates on a **source-destination architecture**:
+- **Source clients**: Original client records from HMIS data sources
+- **Destination clients**: Deduplicated warehouse records (stored in destination data source)
+- **WarehouseClient**: Junction table linking source clients to their destination client
+
+### Key Models
+- **`GrdaWarehouse::Hud::Client`**: Source and destination client records
+- **`GrdaWarehouse::WarehouseClient`**: Links source to destination clients
+- **`GrdaWarehouse::ClientMatch`**: Tracks candidate matches for review
+- **`GrdaWarehouse::ClientSplitHistory`**: Prevents re-merging manually split clients
+- **`GrdaWarehouse::IdentifyDuplicatesLog`**: Tracks algorithm execution statistics
+
+### Data Flow
+```mermaid
+graph TD
+    subgraph DS1["HMIS Data Source 1"]
+        B["Source Client"]
+        B --> E["WarehouseClient"]
+    end
+
+    subgraph DS2["HMIS Data Source 2"]
+        D["Source Client"]
+        D --> F["WarehouseClient"]
+    end
+
+
+    subgraph DSW["Warehouse Data Source"]
+        H["Destination Client<br/>(data_source_id: 99)"]
```

Comment:
This data source id is almost never 99, it is often 1 or 2 though.  Do we want to indicate somewhere that this is not a fixed number?